x/cdp/types/keys.go

```diff
@@ -10,6 +17,159 @@ const (
 	// RouterKey Top level router key
 	RouterKey = ModuleName
 
+	// QuerierRoute Top level query string
+	QuerierRoute = ModuleName
+
 	// DefaultParamspace default name for parameter store
 	DefaultParamspace = ModuleName
-)
\ No newline at end of file
+)
+
+var sep = []byte(":")
+
+// Keys for cdp store
+// Items are stored with the following key: values
+// - 0x00<cdpOwner_Bytes>: []cdpID
+//    - One cdp owner can control one cdp per collateral type
+// - 0x01<collateralDenomPrefix>:<cdpID_Bytes>: CDP
+//    - cdps are prefix by denom prefix so we can iterate over cdps of one type
+//    - uses : as separator, otherwise the cdp with ID 11 would be selected when iterating over denom with prefix 1
+// - 0x02<collateralDenomPrefix>:<collateralDebtRatio_Bytes>:<cdpID_Bytes>: cdpID
+// - Ox03: nextCdpID
+// - 0x04: debtDenom
+// - 0x05<depositState>:<cdpID>:<depositorAddr_bytes>: Deposit
+// - 0x06<denom>:totalPrincipal
+// - 0x07<denom>:feeRate
+// - 0x08:previousBlockTime
+// - 0x20 - 0xff are reserved for collaterals
```

Comment:
Remove this comment? Correct me if I'm wrong but collateral types are assigned byte ids, which as used in store keys, but their namespace doesn't conflict with the other store keys. So they can start at 0x00.